### PR TITLE
check for fuchsia

### PR DESCRIPTION
**Pull request by dnfield**
Make sure the Fuchsia artifacts are available before rolling.

This can happen if someone manually rolls the engine, or if the roller gets misconfigured to not look for fuchsia.stamp

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -70,6 +70,12 @@ task:
       script:
         - dart --enable-asserts ./dev/bots/analyze.dart
 
+    - name: fuchsia_precache-linux # linux-only
+      only_if: "changesInclude('.cirrus.yml', 'bin/internal/engine.version')"
+      script:
+        - flutter precache --fuchsia --no-android --no-ios
+        - flutter precache --flutter_runner --no-andorid --no-ios
```
Comment (xster) [inline .cirrus.yml]: s/andorid/android

Comment (dnfield) [reply .cirrus.yml]: thanks, done

**Comment by dnfield** [comment]
It looks like this precache doesn't fail - I would have expected it to.  What gives?

**Comment by xster** [comment]
Weird that this test ran on engine c4229bfbba whereas head engine version is d60f298d9e7755b8f8204646e7ff03a846f5436c which is the one with the missing fuchsia thing

**Review by iskakaushik — APPROVED** [review]
LGTM

**Comment by dnfield** [comment]
Ok, now it's red as expected.

**Comment by dnfield** [comment]
Landing on red, to make the tree, because the tree is broken and no one knows right now.

**Pull request merged**